src/Microsoft.AspNet.Mvc.Abstractions/ModelBinding/ModelValidationNode.cs

```diff
@@ -0,0 +1,75 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using Microsoft.Framework.Internal;
+
+namespace Microsoft.AspNet.Mvc.ModelBinding
+{
+    /// <summary>
+    /// Captures the validation information for a particular model.
+    /// </summary>
+    public class ModelValidationNode
+    {
+        /// <summary>
+        /// Creates a new instance of <see cref="ModelValidationNode"/>.
+        /// </summary>
+        /// <param name="key">The key that will be used by the validation system to log <see cref="ModelState"/>
```

Comment:
log -> find corresponding (could leave out "corresponding" but "log" is misleading)